public/lib/behaviors/smart-merge.js

```diff
@@ -1,15 +1,15 @@
 import canMap from 'can-connect/can/map/map';
 import smartMerge from 'can-connect/helpers/map-deep-merge';
-import canBatch from 'can-event/batch/batch';
+import queues from 'can-queues';
```

Comment:
`can-event/batch/` has been replaced by the new queueing system in `can-queues`.